docs/root/start/quick-start/run-envoy.rst

```diff
@@ -282,6 +345,20 @@ For invalid configuration the process will print the errors and exit with ``1``.
          [2020-11-08 12:36:06.549][11][info][config] [source/server/configuration_impl.cc:121] loading stats sink configuration
          configuration 'my-envoy-config.yaml' OK
 
+   .. tab:: Docker, Windows Container
+
+      .. substitution-code-block:: PowerShell
+
+         PS> docker run --rm -it
+         -v "$PWD\:`"C:\envoy-configs`""
+         -p '9901:9901'
+         -p '10000:10000'
+         'envoyproxy/|envoy_windows_docker_image|'
+            --mode validate
+            -c 'C:\envoy-configs\my-envoy-config.yaml'
+
+         configuration 'my-envoy-config.yaml' OK
```

Comment:
should this be indented ? 

it is in every other case in this PR.

also, i guess with powershell you dont need a trailing `\` to prevent the command finishing

Reply:
fixed in 96aa0b05d9f83941cd1a072eeeaa3d5badc77997

Reply:
this one still isnt indented - at least not like the others

Reply:
I don't follow, this is the code I have. Which part is not indented?

```
   .. tab:: Docker (Linux)

      .. substitution-code-block:: console

         $ docker run --rm \
               -v $(pwd)/my-envoy-config.yaml:/my-envoy-config.yaml \
               envoyproxy/|envoy_docker_image| \
                   --mode validate \
                   -c my-envoy-config.yaml
         [2020-11-08 12:36:06.543][11][info][main] [source/server/server.cc:583] runtime: layers:
         - name: base
           static_layer:
             {}
         - name: admin
           admin_layer:
             {}
         [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:95] loading tracing configuration
         [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:70] loading 0 static secret(s)
         [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:76] loading 1 cluster(s)
         [2020-11-08 12:36:06.546][11][info][config] [source/server/configuration_impl.cc:80] loading 1 listener(s)
         [2020-11-08 12:36:06.549][11][info][config] [source/server/configuration_impl.cc:121] loading stats sink configuration
         configuration 'my-envoy-config.yaml' OK

   .. tab:: Docker (Windows)

      .. substitution-code-block:: PowerShell

         PS> docker run --rm -it
         -v "$PWD\:`"C:\envoy-configs`""
         -p '9901:9901'
         -p '10000:10000'
         'envoyproxy/|envoy_windows_docker_image|'
            --mode validate
            -c 'C:\envoy-configs\my-envoy-config.yaml'

         configuration 'my-envoy-config.yaml' OK
```

Reply:
see comments above

Reply:
apologies if i wasnt very clear where i put my comment

Reply:
I figured it out, the comment was helpful. It is that whitespace is confusing sometimes the brain just skips it and you have a hard time figuring out the difference.